component/attr_cache/attr_cache.go

```diff
@@ -354,10 +374,17 @@ func (ac *AttrCache) ReadDir(options internal.ReadDirOptions) (pathList []*inter
 			// merge directory cache into the results
 			var numAdded int // prevent shadowing pathList in following line
 			pathList, numAdded = ac.addDirsNotInCloudToListing(options.Name, pathList)
-			log.Trace("AttrCache::ReadDir : %s +%d from cache = %d",
+			log.Info("AttrCache::ReadDir : %s +%d from cache = %d",
 				options.Name, numAdded, len(pathList))
 		}
 	}
+
+	// values should be returned in ascending order by key, without duplicates
+	// sort
+	slices.SortFunc[[]*internal.ObjAttr, *internal.ObjAttr](pathList, func(a, b *internal.ObjAttr) int {
```

Comment:
Nice use of the slices package!

Reply:
I read that this is faster than `sort.Sort`, and I was using this difficult generic thing for removing duplicates below, so I figured why not! 😅